### PR TITLE
Standardize logging

### PR DESCRIPTION
**Pull request by labrys**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/metadata/media_browser.py [hunk 1]
```diff
@@ -174,12 +164,11 @@ def get_season_poster_path(show_obj, season):
...
 
         if not season_dir:
-            logger.log(u'Unable to find a season directory for season {season_num}'.format
-                       (season_num=season), logger.DEBUG)
+            log.debug(u'Unable to find a season directory for season {0}', season)
             return None
 
-        logger.log(u'Using {path}/folder.jpg as season dir for season {season_num}'.format
-                   (path=season_dir, season_num=season), logger.DEBUG)
+        log.debug(u'Using {path}/folder.jpg as season directory for season {x}',
```
Comment (p0psicles) [inline medusa/metadata/media_browser.py]: No reason the use variables like x here.

File: medusa/metadata/media_browser.py [hunk 2]
```diff
@@ -217,12 +206,11 @@ def get_season_banner_path(show_obj, season):
...
         if not season_dir:
-            logger.log(u'Unable to find a season directory for season {season_num}'.format
-                       (season_num=season), logger.DEBUG)
+            log.debug(u'Unable to find a season directory for season {0}', season)
             return None
 
-        logger.log(u'Using {path}/banner.jpg as season dir for season {season_num}'.format
-                   (path=season_dir, season_num=season), logger.DEBUG)
+        log.debug(u'Using {path}/banner.jpg as season directory for season {x}',
+                  {'path': season_dir, 'x': season})
```
Comment (p0psicles) [inline medusa/metadata/media_browser.py]: Same

File: medusa/metadata/media_browser.py [hunk 3]
```diff
@@ -392,10 +380,12 @@ def _ep_data(self, ep_obj):
             try:
                 my_ep = my_show[ep_to_write.season][ep_to_write.episode]
             except (IndexerEpisodeNotFound, IndexerSeasonNotFound):
-                logger.log(u'Unable to find episode {ep_num} on {indexer}... '
-                           u'has it been removed? Should I delete from db?'.format
-                           (ep_num=episode_num(ep_to_write.season, ep_to_write.episode),
-                            indexer=indexerApi(ep_obj.show.indexer).name))
+                log.info(
+                    u'Unable to find episode {x} on {indexer}... has it been removed? Should I delete from db?', {
+                        'x': episode_num(ep_to_write.season, ep_to_write.episode),
```
Comment (p0psicles) [inline medusa/metadata/media_browser.py]: Same

**Comment by labrys** [comment]
@p0psicles please update review

**Pull request merged**